assets/compat.ini

```diff
@@ -1231,6 +1233,34 @@ UCKS45124 = true
 UCJS10104 = true
 NPJG00047 = true
 
+[ForceLowerResolutionForEffectsOff]
+# Some games really don't work with this. Ratchet & Clank looks terrible.
+UCUS98633 = true
+UCAS40145 = true
+UCES00420 = true
+UCJS10052 = true
+UCKS45048 = true
+UCES00420 = true
+UCJS18030 = true
+UCJS18047 = true
+NPJG00015 = true
+
+# The various Tantalus games will not work with this.
+
+# Cars Race-o-rama
+ULUS10428 = true
+ULES01333 = true
+
+# MX vs ATV Reflex
+ULES01375 = true
+ULUS10429 = true
+
+# Spongebob: The Yellow Avenger
+ULES00280 = true
+ULUS10092 = true
+
+# Also, it's a really bad idea in Kurohyou

```

Comment:
Was this meaning to include Kurohyou or just not hoping for people to use it?

I anticipate there will be some game that both uses buffers for bloom (where ghosting motivates using this setting) and for depth effects (where using this setting would break things.)

-[Unknown]

Reply:
Oh, forgot to add the IDs somehow. 

Heh, yeah, might need more complicated heuristics for such games.